packages/core/addon/components/dropdown-date-picker.ts

```diff
@@ -9,11 +9,15 @@
  *       @dateTimePeriod="day"
  *   />
  */
-import Component from '@ember/component';
-import layout from '../templates/components/dropdown-date-picker';
-import { layout as templateLayout } from '@ember-decorators/component';
+import Component from '@glimmer/component';
+import { Moment } from 'moment';
 
-@templateLayout(layout)
-class DropdownDatePicker extends Component {}
+type Args = {
+  date: Moment;
+  dateTimePeriod: string;
```

Comment:
`import type { Grain } from 'navi-data/utils/date';`
```suggestion
  dateTimePeriod: Grain;
```